internal/util/error.go

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018 The Service Manager Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package util
+
+import "errors"
+
+var (
+	LoginValidationError     = errors.New("when using client credentials flow, required either client_id & client_secret or client_id & cert & key")
+	MTLSKeyPairCreationError = errors.New("error creating x509 keypair from client cert file and client key file")
```

Comment:
ua review

Reply:
`MTLSKeyPairCreationError` - removed, returning the original error
`LoginValidationError` - moved to the package as `validationError`